app/src/main/java/it/quip/android/adapter/NotificationBaseViewHolder.java

```diff
@@ -43,6 +45,20 @@ public void onClick(View v) {
             }
         });
 
+        timestampText.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+
+                Context context = mAdapter.getContext();
+                Notification notification = new Notification.with(context)
+                        .body("Jean Claude Van Como deposited some darkness in @sfsewers thangs")
+                        .sender(User.getUserForSession())
```

Comment:
This shoud be `QuipitApplication.getCurrentUser()`. `getUserForSession` is blocking and is only used right after the login